src/main/java/Xavier.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class Xavier {
+    public static final String LINE = "_________________________________________________________________";
+    static Scanner input = new Scanner(System.in);
+    public static void main(String[] args) {
+        TaskManager taskManager = new TaskManager();
+        printWelcomeMessage();
+        while (true) {
+            String command = input.nextLine();
+            System.out.println(LINE);
+            if (command.equals("bye")) {
+                exit();
+                return;
+            }
+            else {
+                taskManager.handleCommand(command);
+            }
+            System.out.println(LINE);
+        }
+    }
+
+    private static void printWelcomeMessage() {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm " + "Xavier");
```

Comment:
Might be better to utilize one continuous string to print if the text inside is not dynamic (i.e.  being read from a variable), such as:

"Hello! I'm Xavier"